{"name": "Employee User", "status": "passed", "description": "Verify User is not able to Login with Invalid credentials", "attachments": [{"name": "log", "source": "b78f21f5-a31b-4649-a0ac-c03aefd72d2c-attachment.txt", "type": "text/plain"}], "start": 1589016024644, "stop": 1589016037115, "uuid": "beca52ee-27e9-4632-a269-d9df75fc0975", "historyId": "57f7ed73470ce8d4aacc4aec2dc80c63", "testCaseId": "e94f0024c374825736805ac54bd2db4c", "fullName": "tests.home.test_login.TestLogin#test_invalidLogin", "labels": [{"name": "story", "value": "HCMAT-54574"}, {"name": "severity", "value": "minor"}, {"name": "parentSuite", "value": "tests.home"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "del1-lhp-N01966"}, {"name": "thread", "value": "37036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.home.test_login"}]}